dashboard/lib/policies/child_account.rb

```diff
@@ -76,7 +76,10 @@ def self.show_cap_state_modal?(user)
   # Checks if a user is affected by a state policy but was created prior to the
   # policy going into effect.
   def self.user_predates_policy?(user)
-    parent_permission_required?(user) && user.created_at < STATE_POLICY[user.us_state][:start_date]
+    parent_permission_required?(user) && (
+      user.created_at < STATE_POLICY[user.us_state][:start_date] ||
```

Comment:
`STATE_POLICY[user.us_state][:start_date]` will raise `undefined method `[]' for nil:NilClass` error if no policy